src/ios/NativePageTransitions.m

```diff
@@ -718,9 +718,9 @@ - (UIImage*) grabScreenshot {
 }
 
 - (BOOL) loadHrefIfPassed:(NSString*) href {
-  UIWebView *uiwebview = nil;
-  if ([self.webView isKindOfClass:[UIWebView class]]) {
-    uiwebview = ((UIWebView*)self.webView);
+  WKWebView *uiwebview = nil;
```

Comment:
Could we add a fallback here to still use UIWebView on older versions of iOS for backward compatibility?

Reply:
Apple deny upload if uiwebview is present in code, so you can use versioning on NPM